torchvision/prototype/features/_image.py

```diff
@@ -327,3 +327,9 @@ def gaussian_blur(self, kernel_size: List[int], sigma: Optional[List[float]] = N
 
         output = _F.gaussian_blur_image_tensor(self, kernel_size=kernel_size, sigma=sigma)
         return Image.new_like(self, output)
+
+    def normalize(self, mean: List[float], std: List[float], inplace: bool = False) -> Image:
```

Comment:
I wouldn't put the normalize in the Image object. This is because the moment a picture is normalized, we lose information about its max value from within the object which is necessary for many transforms.